packages/flutter_test/test/view_test.dart

```diff
@@ -443,3 +451,10 @@ class _FakeFlutterView implements FlutterView {
     return null;
   }
 }
+
+class _FakeDisplay implements TestDisplay {
```

Comment:
same nit about `extends Fake`

Reply:
Updated! I hadn't seen `Fake` before, this is neat!